Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -437,6 +437,17 @@ open class ChatMessageListVC: _ViewController,
         navigationController?.present(alert, animated: true)
     }
 
+    public func jumpToMessage(_ message: ChatMessage, withHighlight: Bool) {
```

Comment:
maybe `highlighted` instead of `withHighlight`?

Reply:
I also though about `shouldHighlight` 🤔 But I guess `highlighted` works as well... but maybe it is not very clear that it is a boolean?

Reply:
`shouldHighlight` is the best option imo.